assets/src/edit-story/components/carousel/carouselList.js

```diff
@@ -49,23 +49,11 @@ const PageList = styled(Reorderable).attrs({
       overflow-x: visible;
       overflow-x: overlay;
       justify-content: flex-start;
-      /*
-       * These overrides are an exception - generally scrollbars should all
-       * look the same. We do this only here because this scrollbar is always visible
-       * if scroll is possible.
-       */
-      scrollbar-color: ${({ theme }) => theme.DEPRECATED_THEME.colors.bg.v10}
-        ${({ theme }) => theme.colors.bg.primary} !important;
-
-      &::-webkit-scrollbar-track {
-        background: ${({ theme }) => theme.colors.bg.primary} !important;
-      }
-
-      &::-webkit-scrollbar-thumb {
-        border: 2px solid ${({ theme }) => theme.colors.bg.primary} !important;
-        border-top-width: 3px !important;
-      }
     `};
+
+  ::-webkit-scrollbar-thumb {
+    border: 6px solid transparent !important;
```

Comment:
makes the scrollbar smaller for the carousel since the base styling makes the scrollbar cover up the story previews
![small-scrollbar](https://user-images.githubusercontent.com/22185279/109550163-26ce5a80-7a8c-11eb-8b1f-82c536fd3eeb.gif)